eos/signal-pdf.cc

```diff
@@ -125,6 +126,11 @@ namespace eos
                     Options{ },
                     KinematicRange{ "s", 0.0, 11.62, BToDLeptonNeutrino::kinematics_description_s }),
 
+            make_signal_pdf("B->D^*munu::dBR",
```

Comment:
How you make sure that the lepton flavour is really "muons"? 
Is this the default setting?

Reply:
the default lepton in eos is muon.

Reply:
Maybe we can put it there explicitly not relying on the setting of default values,
maybe they get changed one day, although in this case very unlikely

Reply:
so I keep it as it is, or did you mean to include something else here?

Reply:
Yes, could you please put there explicitly that the lepton flavor is muon, I am not
sure how it works, maybe you have to put something in

  Options{{"l", "mu"}}

Reply:
done! and Options{{"l", "mu"}} seems to work.